src/services/integrationRuntimes/format.ts

```diff
@@ -113,47 +110,11 @@ export interface RawAzureIntegrationRuntimeUnion {
   referenceName?: string
 }
 
-const formatParameters = (
-  params: { [propertyName: string]: Record<string, unknown> }
-): AzureLinkedServiceReferencePatameters[] => {
-  if (isEmpty(params)) {
-    return []
-  }
-  return  Object.entries(params).map(([k, v]) => ({
-    id: cuid(),
-    key: k,
-    value: Object.entries(v.value).map(([k2, v2]) => ({
-      id: isObject(v) ? cuid() : `${k2}:${v2}`,
-      key: k,
-      value:
-        (isString(v2) && v2) ||
-        (isArray(v2) &&
-          (v2 as Array<any>)
-            .map(i => (isString(i) && i) || JSON.stringify(i))
-            .join(',')) ||
-        JSON.stringify(v2),
-    })),
-  })) || []
-}
-
-const formatAzureSecret = (
-  secret: RawAzureSecretBaseUnion
-): AzureSecretBaseUnion => {
-  if (isEmpty(secret)) {
-    return {}
-  }
-  return {
-    type: secret?.type,
-    value: secret?.value,
-    store: {
-      type: secret?.store?.type,
-      referenceName: secret?.store?.referenceName,
-      parameters: formatParameters(secret?.store?.parameters),
-    },
-    secretName: Object.values(secret?.secretName ?? {}).join(''),
-    secretVersion: Object.values(secret?.secretVersion ?? {}).join(''),
-  }
-}
+// function useInterfaceToExtractData<ExpectedDataType>(
```

Comment:
is this still needed?

Reply:
No, I can remove it. I was looking to implement a routine that received the resulting type/interface and the object, allowing us to remove unwanted properties using the generated schema interface/type. But this needs some research to do it properly without introducing new bugs.